lib/view_component/compiler.rb

```diff
@@ -237,7 +237,7 @@ def variants_from_inline_calls(calls)
 
     def compiled_template(file_path)
       handler = ActionView::Template.handler_for_extension(File.extname(file_path).gsub(".", ""))
-      template = File.read(file_path)
+      template = File.read(file_path).rstrip
```

Comment:
Super minor, but is there a case where you'd want to keep newlines? e.g. text based mailers

If-so, should we attempt to only strip a single newline off the end of the template?

Reply:
Hmm, perhaps. I think a per-component config option is a better way to go though.